Modules/EMCAL/src/DigitsQcTask.cxx

```diff
@@ -142,21 +179,34 @@ void DigitsQcTask::monitorData(o2::framework::ProcessingContext& ctx)
     //gsl::span<const o2::emcal::Digit> eventdigits(digitcontainer.data() + trg.getFirstEntry(), trg.getNumberOfObjects());
     gsl::span<const o2::emcal::Cell> eventdigits(digitcontainer.data() + trg.getFirstEntry(), trg.getNumberOfObjects());
 
+    //trigger type
+    auto triggertype = trg.getTriggerBits();
+    bool isPhysTrigger = triggertype & o2::trigger::PhT, isCalibTrigger = triggertype & o2::trigger::Cal;
+    std::string trgClass;
+    if (isPhysTrigger)
```

Comment:
we recommend to always add curvy braces.

Reply:
OK, we take this as recommendation :)